addons/website_crm/migrations/11.0.2.0/openupgrade_analysis_work.txt

```diff
@@ -0,0 +1,11 @@
+---Fields in module 'website_crm'---
+website_crm  / website                  / crm_default_team_id (many2one): NEW relation: crm.team
+website_crm  / website                  / crm_default_user_id (many2one): NEW relation: res.users
+
+---XML records in module 'website_crm'---
+DEL crm.team: sales_team.salesteam_website_sales (noupdate)
```

Comment:
This is still not done. Protect the unlink the same way as seen before.

Reply:
Wait, I have seen that this has been now incorporated to `sales_team` module, so you should rename in that module the XML-ID from `website_crm.salesteam_website_sales` to `sales_team.salesteam_website_sales`

Reply:
@pedrobaeza 

done!